src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -63,120 +133,1983 @@ ActorInit En_Kaizoku_InitVars = {
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_PLAYER | AT_TYPE_ENEMY,
+        AC_NONE,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_QUAD,
+    },
+    {
+        ELEMTYPE_UNK4,
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        { 0x00000000, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL | TOUCH_UNK7,
+        BUMP_NONE,
+        OCELEM_NONE,
+    },
     { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
 };
 
-#endif
+static AnimationHeader* sAnimations[] = {
+    &object_kz_Anim_00F8E4, // EN_KAIZOKU_ANIM_0
+    &object_kz_Anim_00EF9C, // EN_KAIZOKU_ANIM_1
+    &object_kz_Anim_00E1C8, // EN_KAIZOKU_ANIM_2
+    &object_kz_Anim_00DBE4, // EN_KAIZOKU_ANIM_3
+    &object_kz_Anim_0058B8, // EN_KAIZOKU_ANIM_4
+    &object_kz_Anim_004860, // EN_KAIZOKU_ANIM_5
+    &object_kz_Anim_002730, // EN_KAIZOKU_ANIM_6
+    &object_kz_Anim_001578, // EN_KAIZOKU_ANIM_7
+    &object_kz_Anim_001390, // EN_KAIZOKU_ANIM_8
+    &object_kz_Anim_0003CC, // EN_KAIZOKU_ANIM_9
+    &object_kz_Anim_000F5C, // EN_KAIZOKU_ANIM_10
+    &object_kz_Anim_00E8BC, // EN_KAIZOKU_ANIM_11
+    &object_kz_Anim_00ED1C, // EN_KAIZOKU_ANIM_12
+    &object_kz_Anim_005644, // EN_KAIZOKU_ANIM_13
+    &object_kz_Anim_00F288, // EN_KAIZOKU_ANIM_14
+    &object_kz_Anim_0043E4, // EN_KAIZOKU_ANIM_15
+    &object_kz_Anim_003A3C, // EN_KAIZOKU_ANIM_16
+    &object_kz_Anim_002BA0, // EN_KAIZOKU_ANIM_17
+    &object_kz_Anim_001E9C, // EN_KAIZOKU_ANIM_18
+};
+
+static u8 sAnimationModes[] = {
+    ANIMMODE_LOOP, // EN_KAIZOKU_ANIM_0
+    ANIMMODE_LOOP, // EN_KAIZOKU_ANIM_1
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_2
+    ANIMMODE_LOOP, // EN_KAIZOKU_ANIM_3
+    ANIMMODE_LOOP, // EN_KAIZOKU_ANIM_4
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_5
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_6
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_7
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_8
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_9
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_10
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_11
+    ANIMMODE_LOOP, // EN_KAIZOKU_ANIM_12
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_13
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_14
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_15
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_16
+    ANIMMODE_LOOP, // EN_KAIZOKU_ANIM_17
+    ANIMMODE_ONCE, // EN_KAIZOKU_ANIM_18
+};
+
+Color_RGBA8 D_80B8AC58 = { 200, 160, 120, 255 };
+Color_RGBA8 D_80B8AC5C = { 130, 90, 50, 255 };
+
+Vec3f D_80B8AC60 = { 0.0f, -1.5f, 0.0f };
+Vec3f D_80B8AC6C = { 0.0f, -0.2f, 0.0f };
+Vec3f D_80B8AC78 = { 300.0f, 0.0f, 0.0f };
+Vec3f D_80B8AC84 = { 0.0f, -3000.0f, 0.0f };
+Vec3f D_80B8AC90 = { 400.0f, 0.0f, 0.0f };
+Vec3f D_80B8AC9C = { 1600.0f, -4000.0f, 0.0f };
+Vec3f D_80B8ACA8 = { -3000.0f, -2000.0f, 1300.0f };
+Vec3f D_80B8ACB4 = { -3000.0f, -2000.0f, -1300.0f };
+Vec3f D_80B8ACC0 = { 1000.0f, 1000.0f, 0.0f };
+
+static TexturePtr sEyeTextures[] = {
+    gKaizokuEyeOpenTex,
+    gKaizokuEyeHalfTex,
+    gKaizokuEyeClosedTex,
+    gKaizokuEyeHalfTex,
+};
```

Comment:
I think a lot of this can move down nearer where it's used, particularly around the limbdraws.